### PR TITLE
Vetting locations

### PR DESCRIPTION
**Pull request by arothuis**
[116602125](https://www.pivotaltracker.com/story/show/116602125)

### CODE REVIEW

**Comment by arothuis** [comment]
@rjkip, could you take a peek?

File: README.md [hunk 1]
```diff
@@ -69,8 +70,8 @@ As a full example:
...
+            "nl_NL": "<p>Beste {{ commonName }},</p>\n\n<p>Bedankt voor het registreren van je token. Je token is bijna klaar voor gebruik. Ga naar de Service Desk om je token te laten activeren. </p>\n<p>Neem aub het volgende mee:</p>\n<ul>\n    <li>Je token</li>\n    <li>Een geldig legitimatiebewijs (paspoort, rijbewijs of nationale ID-kaart)</li>\n    <li>De registratiecode uit deze e-mail</li>\n</ul>\n\n<p style=\"font-size: 150%; text-align: center\">\n    <code>{{ registrationCode }}</code>\n</p>\n\n<p>Service Desk medewerkers die je token kunnen activeren:</p>\n\n{% if vettingLocations is empty %}\n    <p>Er zijn geen Service Desk medewerkers beschikbaar.</p>\n{% else %}\n    <ul>\n        {% for vettingLocation in vettingLocations %}\n            <li>\n                <address>\n                    <strong>{{ vettingLocation.name }}</strong><br>\n                    {{ vettingLocation.location }}<br>\n                    {{ vettingLocation.contactInformation }}\n                </address>\n            </li>\n        {% endfor %}\n    </ul>\n{% endif %}",
```
Comment (rjkip) [inline README.md]: g8 m8

File: src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php [hunk 2]
```diff
@@ -98,7 +98,7 @@ class RaSecondFactor implements JsonSerializable
     public $email;
 
     /**
-     * @ORM\Column(type="stepup_document_number")
+     * @ORM\Column(type="stepup_document_number", nullable=true)
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php]: Would have been nice to include in the commit message why this change was made 0e3949e23b2ebdb30583f30342a44854eda4ac28

**Comment by rjkip** [comment]
So the goal of this PR is to be able send e-mails with one of the two types of vetting locations? If that's the case, :+1:.

File: src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php (hunk 2 above)
Comment (arothuis) [inline src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php]: That would have been nice indeed. I could still add it using some `jedi force push` magic, but I'm pretty sure that is frowned upon.

**Comment by arothuis** [comment]
Yes, the intent here is to show either RA locations or personal RA details.
